sesje/2015-02-11/remiander.py

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
Ten plik chyba do skasowania? To co trzeba mamy w [reminder.md](./reminder.md) i nie ma tam tych zaciemniająco-komplikujących `def`-ów... Chyba, że jest w tym jakiś cel?